### DISCUSSION TITLE
Using with Flow Launcher

### DISCUSSION THREAD

**Discussion by Dudemaan** [valinet/ExplorerPatcher#1623] [Q&A]
I've been attempting to set this up with flow launcher, but whenever I open a file using flow launcher it opens explorer with the ribbon still enabled. The same thing happens when using explorer from cmd, which im pretty sure is doing the same thing. Opening explorer regularly, opening files from the start menu, and opening files using the start command all works properly. Does anyone have a fix for this, or know what the problem is?

**Comment by pyrates999** [comment]
After explore patcher has been installed, symbols are downloaded showing a notification of that.  After you get that notification, you need to then restart file explorer using either task manager of the explorer patcher properties application.
